### ISSUE TITLE
Possibility to pass HTML5DOMDocument::ALLOW_DUPLICATE_IDS to HTML5DOMElement

### ISSUE THREAD

**Issue by cmoeke** [ivopetkov/html5-dom-document-php#41]
When i try to change the innerHTML of an HTML5DOMElement i get the Exception: "A DOM node with an ID value "1" already exists! Pass the HTML5DOMDocument::ALLOW_DUPLICATE_IDS option to disable this check."

Example:
```PHP
<?php
require 'vendor/autoload.php';

$dom = new IvoPetkov\HTML5DOMDocument();
$dom->loadHTML('<!DOCTYPE html>
<html>
<body>
    <dummy id="1">a</dummy>
    <dummy id="2">b</dummy>
    <dummy id="1">c</dummy>
</body>
</html>', \IvoPetkov\HTML5DOMDocument::ALLOW_DUPLICATE_IDS);

$body = $dom->querySelector('body');
$body->innerHTML .= 'd';

echo $dom->saveHTML();
```

The problem is that `\IvoPetkov\HTML5DOMDocument::ALLOW_DUPLICATE_IDS` isn't passed to
```PHP
$tmpDoc->loadHTML('<body>' . $value . '</body>');
```
at [HTML5DOMElement.php Line 100](https://github.com/ivopetkov/html5-dom-document-php/blob/master/src/HTML5DOMElement.php#L100) and [HTML5DOMElement.php Line 111](https://github.com/ivopetkov/html5-dom-document-php/blob/master/src/HTML5DOMElement.php#L111).

Any ideas to solve this problem?
Imho the best option would be to add an setter for innerHTML and outerHTML to pass the options.

**Comment by ivopetkov** [comment]
This was actually a bug in the library (insertHTML() does not throw an error). There should not be such errors when modifying elements too. It's now fixed in v2.2.8.
Thank you for reporting it.

**Comment by cmoeke** [comment]
Thanks for fixing it so fast!
